Source/CarthageKit/Git.swift

```diff
@@ -37,6 +36,19 @@ public struct FetchCache {
 			lastFetchTimes[url] = Date().timeIntervalSince1970
 		}
 	}
+
+    fileprivate static func repositoriesLastFetchTime() -> Date? {
+        do {
+            let attr = try FileManager.default.attributesOfItem(atPath: Constants.Dependency.repositoriesURL.path)
+            return attr[FileAttributeKey.modificationDate] as? Date
```

Comment:
For us relying on file system modification date here — there's far too many antivirus scanners and cloud storage sync tools that continually just modify large swaths of files for me to be comfortable with this 😕

Maybe our own [xattr (extended attribute)](https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man2/setxattr.2.html), could work? I'll try to give some thought to this later…

Reply:
Thank you for your suggestion @jdhealy 🙇 I updated the code - now we are using custom extended attribute instead of modification date attribute 💪 

<img width="583" alt="Screenshot 2019-04-27 at 17 36 54" src="https://user-images.githubusercontent.com/10795657/56851818-aed4fd80-6913-11e9-8e09-ade1c49a3cfa.png">

Could you please have a 👀 🙏